stub-images/vddk-test-vmdk/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+
+container_image(
+    name = "vddk-test-image",
+    entrypoint = "mkdir -p /opt/testing && cp -f /libvddk-test-plugin.so /opt/testing/libvddk-test-plugin.so && qemu-img convert -f raw -O vmdk /cirros.raw /opt/testing/nbdtest.img",
```

Comment:
@nyoxi is there any benefit in setting vddk for this? I thought it only optimizes the transfer from vsphere, no?

Reply:
irrelevant, see other thread